pkgs/tools/inputmethods/uim/default.nix

```diff
@@ -53,6 +54,10 @@ assert withNetworking -> curl != null && openssl != null;
 assert withFFI -> libffi != null;
 assert withMisc -> libeb != null;
 
+let
+  stdenv = if args.stdenv.cc.isGNU then args.gcc13Stdenv else args.stdenv;
+in
+
 stdenv.mkDerivation rec {
   version = "1.8.8";
```

Comment:
Pushed a commit that updates the package, but doesn't help with the VA_COPY issue.